tests/model/test_instance.py

```diff
@@ -0,0 +1,72 @@
+import numpy as np
+import pytest
+from sleap_io.model.instance import (
+    Point,
+    PredictedPoint,
+    Track,
+    Instance,
+    LabeledFrame,
+    PredictedInstance,
+)
```

Comment:
Add these imports to the root [__ init.py__](https://github.com/talmolab/sleap-io/blob/main/sleap_io/__init__.py) file and change to just 
```py
from sleap import (
    Point,
    PredictedPoint,
    Track,
    Instance,
    LabeledFrame,
    PredictedInstance,
)
```